projects/sandbox/train/train/augmentor.py

```diff
@@ -211,28 +211,30 @@ def forward(self, X):
         # sample waveforms and use them to compute
         # interferometer responses
         N = mask.sum().item()
-        responses = self.sample_responses(N, X.shape[-1], psds[mask])
-        responses.to(X.device)
 
-        # perform swapping and muting augmentations
-        # on those responses, and then inject them
-        responses, swap_indices = self.swapper(responses)
-        responses, mute_indices = self.muter(responses)
-        X[mask] += responses
+        if N > 0:
```

Comment:
Remind me when this edge case would be reached? In what instance would we not wan't to inject waveforms?